src/components/Section/VotingSectionTester.css

```diff
@@ -1,17 +1,17 @@
 .VotingSectionTester {
 	display: flex;
 	width: 100%;
-	padding-bottom: 16px;
 	align-items: center;
 	flex-direction: column;
 }
 
 .VotingSectionTester .button.ui.basic.button {
 	border: 1px solid rgba(115, 110, 125, .32);
-	background: #00800017 !important;
+	background: #5c5c6e !important;
```

Comment:
We are using this color in many parts of the code, maybe it would be better to set a variable for it

![image](https://user-images.githubusercontent.com/45410089/168597364-d0f1af8e-50b9-4db3-8424-3e89ba9fbc57.png)